src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
`NODEFS` definitely depends on `FS` - what was the reason for removing it from the list of dependencies?